src/utils/actions.js

```diff
@@ -230,6 +230,77 @@ const actions = [
     },
     requiredInput: ['partial'],
   },
+  {
+    /** *************************************
+     * Execute Automation contacts internal urls to obtain automation
+     * information and execute the automation selected in the property panel in
+     * edit mode.
+     *
+     * ARGS
+     * app - Reference to current app inherited from index.js
+     * automation - the item id of the automation to contact the items api
+     * automationPostData - boolean value. If true, creates a bookmark of the
+     * current selections and sends the resulting bookmark id as an input
+     * parameter to the selected automation.
+    */
+
+    label: 'Execute Automation',
```

Comment:
This will need to use a translation string (similar as the other actions) and that translation string needs to be added to sense-client to be handled in the property panel